composer.json

```diff
@@ -36,9 +36,6 @@
     "autoload": {
         "psr-0": {"IsoCodes": "src/"}
     },
-    "config": {
-        "bin-dir": "bin"
```

Comment:
Makefile's prepared tasks use a few quality-related tools: phpcs, php-cs-fixer, phploc, phpmd, etc. Composer actually (symlink-)install them in that `/bin` dir, so far. If you change that actual vendors bin directory path, they cannot work anymore.